software/sys_controller/ossc/av_controller.c

```diff
@@ -338,8 +338,8 @@ status_t get_status(tvp_input_t input, video_format format)
     if (tc.sync_lpf != cm.cc.sync_lpf)
         tvp_set_sync_lpf(tc.sync_lpf);
 
-    if (memcmp(&tc.col, &cm.cc.col, sizeof(color_setup_t)))
-        tvp_set_fine_gain_offset(&cm.cc.col);
```

Comment:
Seems like another bug was lying here. The code below should be correct:
`if (memcmp(&tc.col, &cm.cc.col, sizeof(color_setup_t)))
    tvp_set_gain_offset(&tc.col);`

Reply:
Yes that works